Source/WebCore/dom/TreeScope.h

```diff
@@ -132,10 +132,15 @@ class TreeScope {
     std::span<const RefPtr<CSSStyleSheet>> adoptedStyleSheets() const;
     ExceptionOr<void> setAdoptedStyleSheets(Vector<RefPtr<CSSStyleSheet>>&&);
 
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
     void addSVGResource(const AtomString& id, RenderSVGResourceContainer&);
+#endif
     void addSVGResource(const AtomString& id, LegacyRenderSVGResourceContainer&);
     void removeSVGResource(const AtomString& id);
```

Comment:
Spotted this: removeSVGResource hasn't been adapted -- it only removes from 'legacyResources' not 'resources'.

Reply:
True! Fixed.